src/main/Validator.java

```diff
@@ -0,0 +1,82 @@
+package main;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class Validator {
```

Comment:
why are all the methods public? you only make a method public if you want someone from outside this class to use it